.github/workflows/checkCI.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on: [pull_request]

```

Comment:
```suggestion
on: [push, pull_request]
```

Reply:
Is direct pushing even possible? Maybe pushing is handled here different compared to MSL. We can do this as you suggest and disable if necessary.